dlme_airflow/utils/dataframe.py

```diff
@@ -11,22 +11,22 @@ def dataframe_from_s3(collection) -> pd.DataFrame:
     @param -- data_path
     """
     data_path = collection.data_path()
+    # TODO: This needs to take creds. See: https://stackoverflow.com/a/45982080/7600626
+    #    -- see how to make this work with localstack so we don't have to hammer S3 for local dev.
```

Comment:
I'd like to leave these comments here for now to capture ideas for future tickets.